pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-deutsche-post.php

```diff
@@ -303,7 +309,7 @@ public function country_available_packet_return(){
 	 * @since [*next-version*]
 	 */
 	public function dhl_order_meta_box() {
-		echo $this->dhl_order_meta_box_table();
+		echo wp_kses_post( $this->dhl_order_meta_box_table() );
```

Comment:
```suggestion
		echo $this->dhl_order_meta_box_table();
```

We should not sanitize HTML content that is made by the plugin itself
<img width="705" alt="image" src="https://github.com/user-attachments/assets/99153064-d733-4a16-abd7-4f544e4fc074">